app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
```

Comment:
u have already inherited Player class from ABC
u don't need to inherit it one more time